code/modules/spells/spell.dm

```diff
@@ -610,6 +610,7 @@
 	SHOULD_CALL_PARENT(TRUE)
 
 	SEND_SIGNAL(src, COMSIG_SPELL_CAST, cast_on)
+	record_featured_object_stat(FEATURED_STATS_SPELLS, name)
```

Comment:
consider a var for if the spell counts for this, "learn spell" is always top 3

Reply:
I'm OK with that, it's really just supposed to be a quick overview you glance at once